x-pack/plugins/ml/public/application/settings/calendars/list/header.tsx

```diff
@@ -0,0 +1,137 @@
+  totalCount: number;
+  refreshCalendars: () => void;
+}
+
+export const CalendarsListHeader: FC<Props> = ({ totalCount, refreshCalendars, isDst }) => {
+  const {
+    services: {
+      docLinks: { links },
+    },
+  } = useMlKibana();
+  const docsUrl = links.ml.calendars;
+  return (
+    <>
+      <MlPageHeader>
+        {isDst ? (
+          <FormattedMessage
+            id="xpack.ml.settings.calendars.listHeader.calendarsTitle"
+            defaultMessage="Daylight savings calendars"
+          />
+        ) : (
+          <FormattedMessage
+            id="xpack.ml.settings.calendars.listHeader.calendarsTitle"
+            defaultMessage="Calendars"
+          />
+        )}
+      </MlPageHeader>
+      <EuiFlexGroup justifyContent="spaceBetween" alignItems="baseline">
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup alignItems="baseline" gutterSize="m" responsive={false}>
+            <EuiFlexItem grow={false}>
+              <EuiTextColor color="subdued">
+                <p>
+                  <FormattedMessage
+                    id="xpack.ml.settings.calendars.listHeader.calendarsListTotalCount"
+                    defaultMessage="{totalCount} in total"
+                    values={{
+                      totalCount,
+                    }}
+                  />
+                </p>
+              </EuiTextColor>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup alignItems="baseline" gutterSize="m" responsive={false}>
+            <EuiFlexItem grow={false}>
+              <EuiButtonEmpty
+                size="s"
+                iconType="refresh"
+                onClick={refreshCalendars}
+                data-test-subj="mlCalendarListRefreshButton"
+              >
+                <FormattedMessage
+                  id="xpack.ml.settings.calendars.listHeader.refreshButtonLabel"
+                  defaultMessage="Refresh"
+                />
+              </EuiButtonEmpty>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSpacer size="m" />
+      <EuiText>
+        <p>
+          <EuiTextColor color="subdued">
+            {isDst ? (
+              <FormattedMessage
+                id="xpack.ml.settings.calendars.listHeader.calendarsDescription"
+                defaultMessage="DST Calendars contain a list of scheduled events for which you do not want to generate anomalies, taking into account daylight saving time shifts that may cause events to occur one hour earlier or later. The same calendar can be assigned to multiple jobs.{br}{learnMoreLink}"
+                values={{
+                  br: <br />,
+                  learnMoreLink: (
+                    <EuiLink href={docsUrl} target="_blank">
+                      <FormattedMessage
+                        id="xpack.ml.settings.calendars.listHeader.calendarsDescription.learnMoreLinkText"
```

Comment:
Should the id here contain `Dst`, e.g `calendarsDstDescription`

Reply:
Updated in https://github.com/elastic/kibana/pull/193605/commits/db72ef0440d4e606a284040d561af0f981647fc5